sys/openbsd/dev_pf.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <sys/types.h>
+include <sys/time.h>
+include <sys/fcntl.h>
+include <sys/ioctl.h>
+include <net/if.h>
+
+include <net/pfvar.h>
+
+resource fd_pf[fd]
+
+openat$pf(fd const[AT_FDCWD], file ptr[in, string["/dev/pf"]], flags flags[open_flags], mode const[0]) fd_pf
+
+ioctl$DIOCKILLSTATES(fd fd_pf, cmd const[DIOCKILLSTATES])
```

Comment:
We add it just to prohibit, right?
What we do for linux in such cases is:
```
_ = DIOCKILLSTATES
```
in some of existing files (e.g. sys.txt), maybe with a comment that we restrict it.
This allows to get access to the const value in init.go.

Otherwise it's more descriptions and anybody reading this actually thinks we wan to test this.

Reply:
> We add it just to prohibit, right?

Correct. Didn't know at that trick; fixed.